{"name": "test_get_search", "status": "broken", "statusDetails": {"message": "TypeError: api_code() got an unexpected keyword argument 'json'", "trace": "@pytest.fixture(scope=\"session\")\n    def login_fixture():\n        data = get_data()['login1']\n        username = data['username']\n        password = data['password']\n        code=data['code']\n>       loginApi.send_code(data)\n\n..\\conftest.py:30: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <api.login_ok.LoginApi object at 0x000001AC0CB45E20>\njson_data = {'code': 2, 'password': 'HM_2023_test', 'username': 'admin'}\n\n    def send_code(self,json_data):\n        \"\"\"\n        获取短信验证码\n        :param json_data:\n        :return:\n        \"\"\"\n>       response = api_util.api_code(json=json_data).json()\nE       TypeError: api_code() got an unexpected keyword argument 'json'\n\n..\\..\\api\\login_ok.py:24: TypeError"}, "start": 1723729112553, "stop": 1723729112553, "uuid": "b592c142-c872-4415-b27e-c3230e89e00e", "historyId": "8f194988b3c67b4451ecf2193e46f567", "testCaseId": "8f194988b3c67b4451ecf2193e46f567", "fullName": "test_case.search_center.test_data_list.Test_search#test_get_search", "labels": [{"name": "severity", "value": "blocker"}, {"name": "epic", "value": "客达天下epic"}, {"name": "feature", "value": "查询课程列表模块"}, {"name": "parentSuite", "value": "test_case.search_center"}, {"name": "suite", "value": "test_data_list"}, {"name": "subSuite", "value": "Test_search"}, {"name": "host", "value": "DESKTOP-SSRE6IE"}, {"name": "thread", "value": "23208-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.search_center.test_data_list"}]}